ports/cef/Cargo.lock

```diff
@@ -118,6 +118,11 @@ version = "0.3.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 
 [[package]]
+name = "bitflags"
```

Comment:
Unfortunately, yes. Worse, 0.5.0 has been released, with a real compat issue (https://github.com/rust-lang-nursery/bitflags/pull/38). I'd prefer just pushing 0.5 through rather than going through all the deps twice.